Makefile

```diff
@@ -11,7 +11,7 @@ BINDATA := modules/{options,public,templates}/bindata.go
 STYLESHEETS := $(wildcard public/less/index.less public/less/_*.less)
 JAVASCRIPTS :=
 
-LDFLAGS += -X "main.Version=$(shell git describe --tags --always | sed 's/-/+/' | sed 's/^v//')"
+LDFLAGS ?= -X "main.Version=$(shell git describe --tags --always | sed 's/-/+/' | sed 's/^v//')"
```

Comment:
This only makes it set LDFLAGS if it's is _not_ set already 

> If you’d like a variable to be set to a value only if it’s not already set, then you can use the shorthand operator ‘?=’ instead of ‘=’

https://www.gnu.org/software/make/manual/html_node/Setting.html#Setting